src/status_im2/contexts/chat/composer/edit/view.cljs

```diff
@@ -30,20 +30,18 @@
     {:size                24
      :icon-only?          true
      :accessibility-label :edit-cancel-button
-     :on-press            (fn []
-                            (utils/cancel-edit-message state)
-                            (rf/dispatch [:chat.ui/cancel-message-edit]))
```

Comment:
Are you sure dispatch event not needed?

Reply:
I added it to `utils/cancel-edit-message`